tests/verified.test.js

```diff
@@ -0,0 +1,63 @@
+import { describe, expect, test } from "@jest/globals";
+import { loadReviewsForMovie } from "../src/movies.js";
+import { apiAdapter } from "../src/reviewList.js";
+
+const API_BASE = "https://plankton-app-xhkom.ondigitalocean.app/api/";
+
+describe("Tests the API for filtering only verified reviews", () => {
+
+    // Tested both things at once to avoid calling the same functions and repeating code to mutch
```

Comment:
While avoiding repetitive code and calling functions as rarely as possible may be desirable when writing program code, but when you're writing tests there are other imperatives, such as keeping it simple and not adding complexity. If you can remove logic and make the test easier to read by doing the same thing in multiple tests, that's usually desirable.